docs/developing-fuzzbench/adding_a_new_benchmark.md

```diff
@@ -35,22 +35,57 @@ powerful but also more work.
 
 ## OSS-Fuzz benchmarks
 
-You can use any existing OSS-Fuzz project as a benchmark. You just need to
-create a single file `oss-fuzz.yaml` in the benchmark directory.
+You can use most existing OSS-Fuzz projects a benchmark. First find out which
```

Comment:
Change the first two sentences to
`You can use most OSS-Fuzz projects as benchmarks. First decide which project and fuzz target you want to use as a benchmark.`